app/styles/main.css

```diff
@@ -791,4 +793,5 @@ section{word-break:break-word;}
   td{text-align:center;}
   .sticky-share{display:flex!important;flex-direction:row;justify-content:center;align-items:center;width:100%;height:50px;bottom:0!important;left:0;top:initial;z-index:20;background-color:white;border-top:1px solid #f0f0f0;}
   .sticky-share div{margin:10px 10px 10px 10px;}
+  .no-mobile-paddings{padding-left:0px; padding-right: 0px;}
```

Comment:
Again, lets remove class `.no-mobile-paddings` and add the class `no-padding` where need be in the html.